e2e/pages/respondToClaimLRspec/defenceType.page.js

```diff
@@ -0,0 +1,47 @@
+const {I} = inject();
```

Comment:
For all the files created under respondToClaimLRspec package, lets follow a consistent pattern -either we add a LRspec suffix or we don't. currently it is mixed naming.